wolfcrypt/src/random.c

```diff
@@ -2711,6 +2711,10 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
 
 #elif defined(USE_WINDOWS_API)
 
+#ifdef WIN_REUSE_CRYPT_HANDLE
```

Comment:
if it is safe optimisation, can it be enabled unconditionally without build option? If it is not and users must make choice what are consequences here?

Reply:
@redbaron , in order to consider making this default I'll have to do some more testing with multiple threads to confirm there aren't any issues.